packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.scss

```diff
@@ -1,20 +1,61 @@
-.product-payment {
-  .subscription-ready {
 
-    h2 {
-      color: #20123b;
-      font-size: 24px;
-      font-weight: 500;
-    }
 
-    p {
-      color: #0c0c0d;
-      font-size: 16px;
-      margin: 20px 100px;
+    .product-payment {
+      .subscription-ready {
+        display: grid;
+        grid-template-rows: repeat(6, auto);
+
+        .subscription-message {
+          margin-bottom: 5px;
+        }
+
+        h2 {
+          color: #20123b;
+          font-size: 24px;
+          font-weight: 500;
+        }
+
+        p {
+          color: #0c0c0d;
+          font-size: 16px;
+          margin: 20px 100px;
+        }
+
+        a {
+          font-size: 16px;
+        }
+
+        .formBreakBottom {
+          border: solid 1px #d5d4d5;
+          height: 1px;
+          margin-bottom: 1.8em;
+          margin-left: -3em;
+          margin-top: -1em;
+          width: 115%;
+        }
+
+        .formBreakTop {
+          border: solid 1px #d5d4d5;
+          height: 1px;
+          margin-bottom: 1.8em;
+          margin-left: -3em;
+          width: 115%;
+        }
+
+        .expMessage {
+          font-size: 16px !important;
```

Comment:
Why do we need `!important` here, can we override by being more specific with a selector?

Also, my SCSS linter complains with: "SelectorFormat: Selector `expMessage` should be written in lowercase with hyphens", can we change these to lower case dashes?

And, is there any way we can use `px` values here instead of `em`s? Since `em` depends on the parent element, we might get some weirdness here if styles on the parent element change. `px` gives us more control.

Reply:
I made these changes and also changed all the other class names that were camel cased.